rest_framework/schemas/openapi.py

```diff
@@ -47,11 +47,13 @@ def get_paths(self, request=None):
             operation = view.schema.get_operation(path, method)
             # Normalise path for any provided mount url.
             if path.startswith('/'):
+                tags = [path.split('/')[1]]
                 path = path[1:]
             path = urljoin(self.url or '/', path)
 
             result.setdefault(path, {})
             result[path][method.lower()] = operation
+            result[path][method.lower()].update(tags=tags)
```

Comment:
@phuongtai @carltongibson This will not work for nested resources.

Let's assume a restaurant management system. We have restaurants. Each restaurant has branches and each branch has inspections. To retrieve single inspection for a branch, following might be a URL:

**_/restaurants/{restaurant_id}/branches/{branch_id}/inspections/{inspection_id}_**

**Ideally, tags for this path will be `inspections` but your code will generate `restaurants`**

Realtime Example: [Retrieve billing information for a sub-account by zoom.us](https://redocly.github.io/redoc/?url=https://api.apis.guru/v2/specs/zoom.us/2.0.0/swagger.yaml#tag/Billing)